### DISCUSSION TITLE
Fünfte Woche

### DISCUSSION THREAD

**Discussion by MelonigeMelone** [MelonigeMelone/xCalendar#13] [Blogkanal]
# Woche 5

## Neuer Sprint

Wir starten in Sprint 3, Sprint 2 lief ganz gut allerdings wurden nicht alle Tasks erledigt, auf Grund zeitlichen Schwierigkeiten, da 3 von uns Präsentation für Software Engineering vorbereiten mussten, so allerdings Wissen über einzelne Gebiete welche wir brauchen können angereichert haben. 

## Jenkins mit Repository verknüpft

Jenkins wurde nun vollständig mit dem Repository verknüpft. Bei jedem Merge Request wird das Angular Projekt gebuildet und nur bei erfolgreichem Build kann der Merge Request auch gemergt werden.
Bei dem finalen Merge in die Main wird diese erneut gebaut und auf ein Live Test System zu finden, welches ihr hier findet https://dev01.melonigemelone.de dort könnt ihr absofort immer unseren aktuellen Stand anschauen. Zusätzlich könnt ihr auch hier jederzeit den Stand unsere Jenkins Pipelines sehen https://jenkins.melonigemelone.de/job/xCalendar/

## Ersten Code Implementiert

Wir haben den ersten Coden implementiert - was das Frontend angeht. Bevor man unsere App nutzen kann muss man eingeloggt sein. Dementsprechend haben wir uns entschieden als ersten Schritt eine Login Seite zu erstellen. Um eine optische Vorstellung der Seite zu haben folg ein Screenshot dieser.

![image](https://github.com/MelonigeMelone/xCalendar/assets/136441045/9b0dbfd3-e805-47d8-ac49-47ae62b8fd3f)

**Comment by YPuest** [comment]
Hi, finde euren Eintrag schön kurz und Informativ. Gut finde ich außerdem eure Ehrlichkeit zu Beginn, dass ihr in eurem Sprint nicht alle Tasks erledigen konntet.
Mir gefällt euer Design, freuen würde ich mich, wenn es einen Test-Login gäbe, so dass man nicht zwingend einen Account erstellen muss.

**Comment by Christian-2003** [comment]
Hallo zusammen,

ihr habt einen schönen kurzen Blogeintrag geschrieben.

Das Design eures Logins sieht sehr modern aus und gefällt mir gut. Da weiß jeder (potenzielle) Benutzer sofort intuitiv, was zu tun ist.

Ebenfalls gefällt mir eure Entscheidung gut, dass euer Projekt erst bei erfolgreichem Build gemergt werden kann. Das ist meiner Meinung nach sehr sinnvoll und wird sich bestimmt während der Entwicklung auszahlen.

Weiter so, ich freue mich auf euren nächsten Blogeintrag.
